plugins/couchbase/couchbaseembed.cpp

```diff
@@ -512,9 +510,11 @@ namespace couchbaseembed
             unsigned numchars = rtlUtf8Length(strlen(value), value);
             rtlUtf8ToStrX(chars, result, numchars, value);
         }
-
-        NullFieldProcessor p(NULL);
-        rtlStrToStrX(chars, result, p.resultChars, p.stringResult);
+        else
+        {
+            NullFieldProcessor p(NULL);
```

Comment:
Would it make sense to make NullFieldProcessor a class member since it seems to be used a lot?